sed/config/flash_example_config.yaml

```diff
@@ -150,13 +160,27 @@ dataframe:
       dataset_key: "/uncategorised/FLASH.EXP/HEXTOF.DAQ/DLD1/value"
       slice: 4
       dldAuxChannels:
-        sampleBias: 0
-        tofVoltage: 1
-        extractorVoltage: 2
-        extractorCurrent: 3
-        cryoTemperature: 4
-        sampleTemperature: 5
-        dldTimeBinSize: 15
+        sampleBias:
+          slice: 0
+          dtype: float32
```

Comment:
If you treat them that way with "slice", can't you just tread them as individual channels and drop the extra handling altogether?

Reply:
The issue is we already need to slice the dld channels first (See line 161). So it's a nested array. 
Probably np.take can take a list of slice indexes. I can take a look at that.

Reply:
I looked into doing it another way but it makes it way more complex. So for now, I am just keeping this but generalized it to be called subChannels